rosidl_generator_py/package.xml

```diff
@@ -20,11 +20,9 @@
   <buildtool_export_depend>rosidl_typesupport_c</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_interface</buildtool_export_depend>
 
-
   <exec_depend>python3-numpy</exec_depend>
   <exec_depend>rmw_implementation</exec_depend>
   <exec_depend>rmw_implementation_cmake</exec_depend>
-  <exec_depend>rosidl_generator_c</exec_depend>
```

Comment:
The headers from `rosidl_generator_c` are being used by the code generated for messages by this package: https://github.com/ros2/rosidl_python/blob/f19748a1150375a1339c84fbd8a873aa4d0d2ec4/rosidl_generator_py/resource/_msg_pkg_typesupport_entry_point.c.em#L12_L15

So this dependency can't be just removed but needs to be replaced with `rosidl_runtime_c`.

Reply:
replaced with `rosidl_runtime_c`